### ISSUE TITLE
Translation for Dutch

### ISSUE THREAD

**Issue by tijlleenders** [tijlleenders/ZinZen#52]
Make the translations for Dutch.

**Comment by tijlleenders** [comment]
@mallikarai05 I thought with i18next we had separate files per language - so that only that gets downloaded/cached on the users device - but I can't find the files. Are we using a different approach?

**Comment by mallikarai05** [comment]
@tijlleenders yes, there are separate files per language. They are in src/translations/

**Comment by tijlleenders** [comment]
Fantastic, I'll get to it then!

**Comment by tijlleenders** [comment]
Added translation file but accidentally committed to main directly.
I've now activated a protection rule for branch main:
- Require a pull request before merging
All commits must be made to a non-protected branch and submitted via a pull request before they can be merged into a branch that matches this rule.
- Require status checks to pass before merging (I've added 'eslint' and 'cypress-run' GitHub actions)
Choose which [status checks](https://docs.github.com/rest/reference/commits#commit-statuses) must pass before branches can be merged into a branch that matches this rule. When enabled, commits must first be pushed to another branch, then merged or pushed directly to a branch that matches this rule after status checks have passed.
- Require branches to be up to date before merging
This ensures pull requests targeting a matching branch have been tested with the latest code. This setting will not take effect unless at least one status check is enabled
- Include administrators
Enforce all configured restrictions above for administrators.
